server/prisma/migrations/20221222193340_initial_tables/migration.sql

```diff
@@ -0,0 +1,63 @@
+-- CreateTable
+CREATE TABLE "users" (
+    "id" SERIAL NOT NULL,
+    "created_at" TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "username" TEXT NOT NULL,
+    "password" TEXT NOT NULL,
+    "logged_in_at" TIMESTAMPTZ,
+
+    CONSTRAINT "users_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "sessions" (
+    "id" TEXT NOT NULL,
+    "valid" BOOLEAN NOT NULL DEFAULT true,
+    "userId" INTEGER NOT NULL,
+
+    CONSTRAINT "sessions_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "talks" (
+    "id" SERIAL NOT NULL,
+    "created_at" TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "title" TEXT NOT NULL,
+    "description" TEXT NOT NULL,
+    "voted" INTEGER NOT NULL,
+    "owner_id" INTEGER NOT NULL,
+
+    CONSTRAINT "talks_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateTable
+CREATE TABLE "votes" (
+    "id" SERIAL NOT NULL,
+    "created_at" TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "score" INTEGER NOT NULL,
+    "talkId" INTEGER NOT NULL,
+    "owner_id" INTEGER NOT NULL,
+
+    CONSTRAINT "votes_pkey" PRIMARY KEY ("id")
+);
+
+-- CreateIndex
+CREATE UNIQUE INDEX "users_username_key" ON "users"("username");
+
+-- CreateIndex
+CREATE INDEX "users_username_idx" ON "users"("username");
+
+-- CreateIndex
+CREATE INDEX "sessions_userId_idx" ON "sessions"("userId");
+
+-- AddForeignKey
+ALTER TABLE "sessions" ADD CONSTRAINT "sessions_userId_fkey" FOREIGN KEY ("userId") REFERENCES "users"("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
I have two questions here:
- What's the benefit of foreign key? 
- Is it worth using foreign key consider it's performance impact to insert/update/delete?